### PR TITLE
Replace external resources with CDN links

### LINKED ISSUE #359: Replace external resources with CDN links
**Is your feature request related to a problem? Please describe.**
We have to replace all the external resources with CDN links to reduce file sizes and loading times.

**Describe the solution you'd like**
- Replace all the external js scripts and CSS files with CDN links (Except argon files & custom CSS files)
   ex: 
   replace `<link href="assets/vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet">`
   with,
   `<link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">`
- Delete replaced packages from `assets/vendor`

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #359 

## Goals

- Replace external resources with CDN links

## Approach

- Replace all the external js scripts and CSS files with CDN links
- Delete replaced packages from assets/vendor

### Preview Link

http://cdn-links.surge.sh

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/SA.html [hunk 1]
```diff
@@ -87,12 +87,13 @@
 
 <!-- Core -->
 <script src="assets/vendor/jquery/jquery.min.js"></script>
-<script src="assets/vendor/popper/popper.min.js"></script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
 <script src="assets/vendor/bootstrap/bootstrap.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/SA.html]: Add cdn Link to Bootstrap

File: src/main/webapp/SA.html [hunk 2]
```diff
@@ -87,12 +87,13 @@
...
-<script src="assets/vendor/headroom/headroom.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
 <!-- Optional JS -->
 <script src="assets/vendor/onscreen/onscreen.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/SA.html]: Onscreen is no longer needed. Remove this from all pages

File: src/main/webapp/SA.html [hunk 3]
```diff
@@ -87,12 +87,13 @@
...
-<script src="assets/vendor/nouislider/js/nouislider.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
 <script src="assets/vendor/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/SA.html]: Datepicker is no longer needed. Remove this from all pages

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -319,12 +319,13 @@ <h2 class="font-weight-900 text-dark text-roboto text-center">
 
 <!-- Core -->
 <script src="assets/vendor/jquery/jquery.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/index.html]: Add CDN link to jquery (latest version)

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/examples/team.html [hunk 5]
```diff
@@ -49,9 +49,9 @@
 <!-- to use the dynamic footer, use this instead of above tag => *<div id="footer"></div>* -->
 
 <!-- Core -->
-<script src="../assets/vendor/jquery/jquery.min.js"></script>
+<script src="../https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 <script src="../assets/vendor/popper/popper.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html]: Add CDN link to popper

File: src/main/webapp/index.html [hunk 6]
```diff
@@ -318,14 +318,15 @@ <h2 class="font-weight-900 text-dark text-roboto text-center">
 </body>
 
 <!-- Core -->
-<script src="assets/vendor/jquery/jquery.min.js"></script>
-<script src="assets/vendor/popper/popper.min.js"></script>
-<script src="assets/vendor/bootstrap/bootstrap.min.js"></script>
-<script src="assets/vendor/headroom/headroom.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/index.html]: Remove extra whitelines

File: src/main/webapp/index.html [hunk 7]
```diff
@@ -318,14 +318,15 @@ <h2 class="font-weight-900 text-dark text-roboto text-center">
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
 <!-- Optional JS -->
-<script src="assets/vendor/onscreen/onscreen.min.js"></script>
-<script src="assets/vendor/nouislider/js/nouislider.min.js"></script>
-<script src="assets/vendor/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/index.html]: Remove extra whiteline

File: src/main/webapp/index.html [hunk 8]
```diff
@@ -318,14 +318,15 @@ <h2 class="font-weight-900 text-dark text-roboto text-center">
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/index.html]: Remove extra whiteline

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/examples/components.html [hunk 9]
```diff
@@ -2822,9 +2822,9 @@ <h4 class="mb-0 font-weight-light">Let's get in touch on any of these platforms.
 
 </footer>
 <!-- Core -->
-<script src="../assets/vendor/jquery/jquery.min.js"></script>
+<script src="../https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
 <script src="../assets/vendor/popper/popper.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/components.html]: Link popper

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Comment by anjula-sack** [comment]
done

File: src/main/webapp/examples/landing.html [hunk 10]
```diff
@@ -702,9 +702,9 @@ <h4 class="mb-0 font-weight-light">Let's get in touch on any of these platforms.
     </div>
   </footer>
   <!-- Core -->
-  <script src="../assets/vendor/jquery/jquery.min.js"></script>
-  <script src="../assets/vendor/popper/popper.min.js"></script>
-  <script src="../assets/vendor/bootstrap/bootstrap.min.js"></script>
+  <script src="../https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
+  <script src="../https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/landing.html]: There are mistakenly added `../`  before some links

File: src/main/webapp/irc.html [hunk 11]
```diff
@@ -87,14 +86,15 @@
 </body>
 
 <!-- Core -->
-<script src="assets/vendor/jquery/jquery.min.js"></script>
-<script src="assets/vendor/popper/popper.min.js"></script>
-<script src="assets/vendor/bootstrap/bootstrap.min.js"></script>
-<script src="assets/vendor/headroom/headroom.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/irc.html:90]: Remove whitespace

File: src/main/webapp/onelive.html [hunk 12]
```diff
@@ -221,14 +221,15 @@ <h1 class="mb-0 text-darker font-weight-lighter">Recommend a speaker</h1>
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
 <!-- Optional JS -->
-<script src="assets/vendor/onscreen/onscreen.min.js"></script>
-<script src="assets/vendor/nouislider/js/nouislider.min.js"></script>
-<script src="assets/vendor/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Remove whitespace

File: src/main/webapp/onelive.html [hunk 13]
```diff
@@ -221,14 +221,15 @@ <h1 class="mb-0 text-darker font-weight-lighter">Recommend a speaker</h1>
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
 <!-- Optional JS -->
-<script src="assets/vendor/onscreen/onscreen.min.js"></script>
-<script src="assets/vendor/nouislider/js/nouislider.min.js"></script>
-<script src="assets/vendor/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Remove whitespace

File: src/main/webapp/onelive.html [hunk 14]
```diff
@@ -221,14 +221,15 @@ <h1 class="mb-0 text-darker font-weight-lighter">Recommend a speaker</h1>
 </body>
 
 <!-- Core -->
-<script src="assets/vendor/jquery/jquery.min.js"></script>
-<script src="assets/vendor/popper/popper.min.js"></script>
-<script src="assets/vendor/bootstrap/bootstrap.min.js"></script>
-<script src="assets/vendor/headroom/headroom.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive.html]: Remove whitespace

File: src/main/webapp/irc.html [hunk 15]
```diff
@@ -87,14 +86,15 @@
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
 <!-- Optional JS -->
-<script src="assets/vendor/onscreen/onscreen.min.js"></script>
-<script src="assets/vendor/nouislider/js/nouislider.min.js"></script>
-<script src="assets/vendor/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/irc.html]: Remove whitespace

File: src/main/webapp/irc.html [hunk 16]
```diff
@@ -87,14 +86,15 @@
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
 <!-- Optional JS -->
-<script src="assets/vendor/onscreen/onscreen.min.js"></script>
-<script src="assets/vendor/nouislider/js/nouislider.min.js"></script>
-<script src="assets/vendor/bootstrap-datepicker/js/bootstrap-datepicker.min.js"></script>
+
```
Comment (jayasanka-sack) [inline src/main/webapp/irc.html]: Remove whitespace

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Comment by jayasanka-sack** [comment]
Thanks @anjula-sack !

**Pull request merged**